Publish minutes of 2022-07-21 meeting (#245)
hober/web-extensions: _minutes/2022-07-21-wecg.md

 * [simeon] Extension store issues are out-of-scope in this CG, but I'm open to hearing from developer concerns right now.
 * [gaurang] I created [a thread in the chromium-extensions google group](https://groups.google.com/a/chromium.org/g/chromium-extensions/c/obZ9t7b8hPk/m/uRm6_aqTAwAJ), Jackie followed up with discussion of Firefox and Edge's stores. The source images have a high quality but are pixelated after uploading to the CWS.
 * [simeon] I'm familiar with this problem & actively discussing it with the CWS team.

[Issue 241](https://github.com/w3c/webextensions/issues/241): Event listeners in Content Scripts and communication between main and isolated CS worlds

 * [jack] I don't believe we can do this (breaking change).
 * [rob] The issue filer is asking to stop firing event listeners in content scripts in response to scripted actions from the web page. This is not a good idea in my opinion; extensions should be prepared to deal with events fired by web pages.
 * [rob] [event.isTrusted](https://developer.mozilla.org/en-US/docs/Web/API/Event/isTrusted) can already be used if the extension really cares that the event was triggered by the user.

>  * [someone] event.isTrusted is only available to events such as click, but not to CustomEvent.

Reviewer: @zombie, did you mention this?

Author: No, someone else did. I don't recall who.

 * [simeon] Would you be open to creating a half-pager or something like that?
 * [tomislav] I'll write a comment on the issue.
 * [rob] We now have three issues about labels. We should consolidate them:
   * https://github.com/w3c/webextensions/issues/205
   * https://github.com/w3c/webextensions/issues/233
   * https://github.com/w3c/webextensions/issues/244 (the one being discussed now)
 * [simeon] Based on initial feedback, we're not going to proceed with my suggestion here.
 * [tomislav] Goal is to have consensus tracking.
 * [rob] If we want to consolidate discussion, suggest using issue 233 as it's the most generic, non-prescriptive one.

[Issue 12](https://github.com/w3c/webextensions/issues/12): allow to retrieve a frameID from an &lt;iframe> element

 * [gaurang] This was raised a few weeks ago with a “Chrome: follow-up” label.
 * [simeon] While reviewing issues with the label, it wasn't clear to me what exactly needs to be followed-up upon. Could you comment on the issue to clarify?
 * [gaurang] I'll create a comment.
 * [simeon] FYI Dave came up a while back to talk about the frameId proposal a while back.
   * [rob] You meant documentId, right?

>    * [simeon] That's right, but I still need to follow up on that one.

Reviewer: ```suggestion
   * [simeon] Right, I got them confused. I also need to follow up on that one.
```

[Issue 162](https://github.com/w3c/webextensions/issues/162): DNR: disable individual static rules

>  * [felipe] Devlin [provided feedback on the current design document](https://docs.google.com/document/d/1NTOTr6iwm0dJbewWjnABmPo6h1QD1mKTpX60s6Klj-8/edit?disco=AAAAc3D154o); wanted to mention that here in case other parties are interested. Also wanted to encourage members to provide feedback on potential use cases for this capability. Delvin is asking if there are more specific use cases where extensions may leverage this capability, such as providing a user-facing control to disable individual static rules.

Reviewer: ```suggestion
 * [felipe] Devlin [provided feedback on the current design document](https://docs.google.com/document/d/1NTOTr6iwm0dJbewWjnABmPo6h1QD1mKTpX60s6Klj-8/edit?disco=AAAAc3D154o); wanted to mention that here in case other parties are interested. Also wanted to encourage members to provide feedback on potential use cases for this capability. Devlin is asking if there are more specific use cases where extensions may leverage this capability, such as providing a user-facing control to disable individual static rules.
```
